pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -577,12 +575,10 @@ func (cm *containerManagerImpl) Start(ctx context.Context, node *v1.Node,
 	}
 
 	// Initialize memory manager
-	if utilfeature.DefaultFeatureGate.Enabled(kubefeatures.MemoryManager) {
-		containerMap, _ := buildContainerMapAndRunningSetFromRuntime(ctx, runtimeService)
-		err := cm.memoryManager.Start(memorymanager.ActivePodsFunc(activePods), sourcesReady, podStatusProvider, runtimeService, containerMap)
-		if err != nil {
-			return fmt.Errorf("start memory manager error: %v", err)
-		}
+	containerMap, _ = buildContainerMapAndRunningSetFromRuntime(ctx, runtimeService)
```

Comment:
we can remove this line and reuse what we computed on line 561 above

Reply:
Done. Thanks!